app/controllers/api/storage.php

```diff
@@ -652,15 +756,17 @@
     ->label('sdk.methodType', 'location')
     ->param('bucketId', null, new UID(), 'Storage bucket unique ID. You can create a new storage bucket using the Storage service [server integration](/docs/server/storage#createBucket).')
     ->param('fileId', '', new UID(), 'File unique ID.')
+    ->inject('request')
     ->inject('response')
     ->inject('dbForInternal')
-    ->action(function ($bucketId, $fileId, $response, $dbForInternal) {
+    ->action(function ($bucketId, $fileId, $request, $response, $dbForInternal) {
```

Comment:
same comments that are relevant for `/view` are also relevant here in `/download`